jarbas/core/fields.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime
+
+from rows.fields import DateField
+from rows.fields import IntegerField as RowsIntegerField
```

Comment:
I'd rather use `import rows.fields` and then inherit from `rows.fields.IntegerField`.